src/queries.js

```diff
@@ -18,9 +34,19 @@ const RaMQuery = `query RaM {
   }`;
 
 export function getSeriesInfo() {
-    axios({ url: 'https://rickandmortyapi.com/graphql', method: 'post', data: { query: RaMQuery } })
-    .then(renderEpisodeButtons)
+    axios({ url: 'https://rickandmortyapi.com/graphql', method: 'post', data: { query: seriesInfo } })
+    .then(setTitleDescription)
     .catch(erro => console.log(erro));
 }
 
-export default RaMQuery; 
\ No newline at end of file
+export function getButtonInfo() {
+  axios({ url: 'https://rickandmortyapi.com/graphql', method: 'post', data: { query: buttonsInfo } })
+  .then(renderEpisodeButtons)
+  .catch(erro => console.log(erro));
+}
+
+export function getCardInfo() {
```

Comment:
this is a nicer separation of the data fetching :)
But you are still querying all data before. the user even clicks the button.
I think you should do in THIS PR the following:
1. Stop querying all episode information that is shown only in the card before the user clicks it
2. When a user clicks the button, add a "Loading-State" inside the card to inform the user, the request is being made to get the information (you can add a simple "Loading card" message or a spinner component (which is more sophisticated https://projects.lukehaas.me/css-loaders/)
3. Replace the "loading-state" with the content of the request